cmd/helm-migrate-values/root.go

```diff
@@ -111,7 +111,14 @@ func newRunner(actionConfig *action.Configuration, flags *pflag.FlagSet, setting
 		if release != nil {
 			log.Debug("Release is using chart: %s", release.Chart.Metadata.Name)
 			log.Debug("Release is currently on chart version: %s", release.Chart.Metadata.Version)
-			log.Debug("Release has the values: %s", release.Config)
+
+			if release.Config != nil && log.IsDebug {
```

Comment:
```suggestion
			if release.Config != nil {
```
log.Debug performs the log level check internally

Reply:
I added the check so we don't have to marshal the values every time even when we're not using them.

Reply:
Oh ok. But that's _probably_ premature optimisation 😄. I doubt the yaml.Marshal is very expensive, especially given that it's only run once during the whole program execution and the release.Config is not that large normally

Reply:
You are of course correct about the premature nature of this 😁